README.md

```diff
@@ -1,13 +1,229 @@
 # CHINESE MENU GENERATOR-BACKEND
 
-## ERD
+#### Team Building API Exercise Using NodeJS/ExpressJS
+
+<hr style="border: solid 1px"/>
+
+## Table of Content
+
+- [Summary](#summary)
+- [Authors](#authors)
+- [Goals](#goals)
+  - [Project First Itearation Goals](#project-first-iteration-goals)
+  - [Extension Goals for Future Iterations](#extension-goals-for-future-iterations)
+- [Stack](#stack)
+- [Install Locally](#install-locally)
+- [Schema](#schema)
+- [Swagger](#swagger)
+- [Endpoints](#endpoints)
+  - [Postman](#postman)
+  - [User Endpoints](#user-endpoints)
+  - [Menu Endpoints](#menu-endpoints)
+  - [Recipe Endpoints](#recipe-endpoints)
+
+<hr style="border: solid 1px"/>
+
+## Summary
+
+The main goal of this exercise was to build an API that in NodeJS and ExpressJS that uses TypeScript including tests - Jest - and documentation - swagger.
+
+As a secondary goal was to learn how to work together so we can successfully complete larger projects in the future. To achieve this we drew from each other's work and life experience and we were able to establish patterns that worked well for us and facilitated fluid communication and standards clarity.
+
+The main technical challenge was to incorporate TypeScript since we are both relatively new to building APIs with NodeJS and TypeOrm.
+
+## Authors
+
+<div align="center">
+
+<img src="https://avatars.githubusercontent.com/u/115114235?v=4" width="150px" />
+
+#### Lingling Gan
+
+Github: [github.com/MandyGan](https://github.com/MandyGan)
+
+LinkedIn: [linkedin.com/in/lingling-gan/](https://www.linkedin.com/in/lingling-gan/)
+
+<img src="https://avatars.githubusercontent.com/u/22056578?v=4" width="150px" />
+
+#### Nico Rithner
+
+Github: [github.com/nicorithner](https://github.com/nicorithner) 
+
+LinkedIn: [linkedin.com/in/nicorithner/](https://www.linkedin.com/in/nicorithner/)
+
+</div>
+
+## Goals
+
+### Project First Iteration Goals
+
+- A crud a API with Users, Menus, and Recipes
```

Comment:
a CRUD (Create, Read, Update, Delete) API with Users, Menus, and Recipes?

Reply:
✅